package.json

```diff
@@ -27,18 +27,18 @@
     "@iconify-json/game-icons": "^1.1.9",
     "@iconify-json/heroicons": "^1.1.20",
     "@iconify-json/ph": "^1.1.12",
-    "@iconify-json/simple-icons": "^1.1.99",
+    "@iconify-json/simple-icons": "^1.1.100",
     "@iconify-json/streamline": "^1.1.8",
     "@nuxt/fonts": "^0.3.0",
-    "@nuxt/image": "^1.5.0",
-    "@sidebase/nuxt-auth": "^0.7.1",
+    "@nuxt/image": "^1.6.0",
```

Comment:
Consider pinning the major version of critical dependencies to avoid automatic updates to potentially incompatible versions. [medium]

Reply:
I updated these deps manually. The goal is to stay current to the latest major releases until closer to launch.